tests/scripts/create_coverage_report.py

```diff
@@ -35,8 +35,12 @@ def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
         lcov_list.append("-a")
         lcov_list.append(f"{merge_dir}/{file_path.name}")
 
+    # Append unit tests code coverage results
+    lcov_list.append("-a")
+    lcov_list.append("/var/tmp/bluechi-coverage/unit-test-results/coverage.info")
+
     # Skip creating coverage report when no .info files are found
-    if len(lcov_list) == 1:
+    if len(lcov_list) == 3:
```

Comment:
If no .info file is found `len(lcov_list) ==1`. And if only one is found, then its 3 - which is also invalid, right? So shouldn't it be then `len(lcov_list) <= 3`?

Reply:
Previously the default length of the list was 1 because of https://github.com/eclipse-bluechi/bluechi/blob/main/tests/scripts/create_coverage_report.py#L30

This patch adds always 2 additional items:

```
lcov_list.append("-a")
lcov_list.append("/var/tmp/bluechi-coverage/unit-test-results/coverage.info")
```

That's why I've increased to 3, but it doesn't matter if we use `==3` or `<=3`, the result should be the same